Resources/Private/Javascript/Components/DefinitionList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import { classList } from '../Helper/utilities';
+
+const DefinitionList = ({ children, modifier }) => {
+  return (
+    <dl className={classList({
+      'c-definition-list': true,
+      ['c-definition-list--' + modifier]: modifier 
+    })}>
+      {children}
+    </dl>
+  );
+};
+
+export const DefinitionItem = ({ label, value, modifier }) => {
+  return (
+    <div className={classList({
+      'c-definition-list__item': true,
+      ['c-definition-list__item--' + modifier]: modifier 
```

Comment:
To be honest, I'm not sure if this is a good way to use BEM (and what i mean is: i really dont know). In my eyes a modifier should be set to change appearance, state or behavior on a comprehensible way (like color or size) and should not be used to give things names. Maybe thats what mixes should be used for? Or maybe BEM is not the right way to go here?

Reply:
Will be discussed and maybe fixed in later improvements